src/docs/product/releases/health/release-details.mdx

```diff
@@ -4,13 +4,51 @@ sidebar_order: 50
 description: "Learn more about the details of individual releases."
 ---
 
-The Release Details page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, and commit author breakdowns.
+The **Release Details** page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, adoption graphs, and commit author breakdowns.
 
-![View of the release details page showing an overview graph, files changed, and other details.](release_details.png)
+![View of the release details page showing a release comparison graph, issues, adoption, and other details.](release_details.png)
```

Comment:
It seems like all of these features that you're describing are early adopter features. If so, and if you're hoping to merge this before those features go GA, then you'll need an EA note:

```suggestion
![View of the release details page showing a release comparison graph, issues, adoption, and other details.](release_details.png)

<Note>

The following features are available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

```